extide/gradle/test/unit/src/org/netbeans/modules/gradle/api/execute/RunUtilsTest.java

```diff
@@ -70,4 +72,15 @@ public void testIncludeOpenProjects2() throws Exception {
         assertEquals(0, params.size());
     }
 
+    public void testEvaluateArgs1() throws Exception {
+        FileObject a = createGradleProject("projectA",
+                "apply plugin: 'java'\n", "");
+        Project prjA = ProjectManager.getDefault().findProject(a);
+        ProjectTrust.getDefault().trustProject(prjA);
+        openProject(a);
+        String[] args = RunUtils.evalueteActionArgs(prjA, null, "Project: ${projectPath}${projectName}", Lookups.singleton(prjA));
```

Comment:
Typo in `evalueteActionArgs`.

Reply:
Thanks for spotting the typo!
Well, I do not know if this support ```java.lsp.server``` in any way. That would be your call to decide.